packages/js/e2e-environment/bin/scaffold.js

```diff
@@ -24,12 +24,24 @@ const {
 } = require( '../utils/scaffold' );
 
 const args = process.argv.slice( 2 );
-const [ command, packageName ] = args;
+const command = args[0];
+let packageName = null;
+
+if ( args[1] && args[1].substr(0, 2) !== '--' ) {
```

Comment:
Small nitpick with the spacing.

```suggestion
if ( args[ 1 ] && args[ 1 ].substr( 0, 2 ) !== '--' ) {
```
Using a linter can help with this